packages/zosfiles/src/api/methods/copy/Copy.ts

```diff
@@ -57,14 +56,12 @@ export class Copy {
         const payload: any = {
             "request": "copy",
             "from-dataset": {
-                dsn: fromDataSetName,
-                member: fromMemberName,
+                dsn: options.fromDataSet.dataSetName,
+                member: options.fromDataSet.memberName,
             },
+            ...options
         };
-
-        if (options.replace !== undefined) {
-            payload.replace = options.replace;
-        }
+        delete payload.fromDataSet;
```

Comment:
@liuxh0, this is kind of what I was looking for. Rather than deleting part of the object, maybe something more generic that can apply to all the [Utilities provided by ZOSMF](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.1.0/com.ibm.zos.v2r1.izua700/IZUHPINFO_API_PutDataSetMemberUtilities.htm) around this.
https://www.ibm.com/support/knowledgecenter/SSLTBW_2.1.0/com.ibm.zos.v2r1.izua700/IZUHPINFO_API_PutDataSetMemberUtilities.htm

For example, an interface that looks very much like the table on the link above : )
`IDataSet`
```
dsn: string,
member?: string,
volser?: string,
alias?: boolean
```

`IFileDescription`
```
filename: string,
type?: "binary|executable|text"
```

`IDataSetUtilities.ts`
```
request: string,
"from-dataset": IDataSet,
enq: string,
"from-file"?: IFileDescription,
replace?: boolean
```

Reply:
Again, I believe this PR is good as it is for now :) 
We can definitely address this interface typing at a later time : )

Reply:
@zFernand0 , thanks for your review. I'm happy to hear that this is what you want. 
I also think it is much better than deleting the object property. I was just not sure if I should change `IDataSet` interface since it also changes other parts. 
If it is okay, I would like to change it in another PR, to keep each PR's single responsibility.
And I also need your help to see if the system tests pass. Thanks!